tables-test-fixtures/build.gradle

```diff
@@ -71,6 +71,7 @@ shadowJar {
       // e.g. java.lang.NoClassDefFoundError: openhouse/relocated/com/google/protobuf/GeneratedMessageV3
       // when users use fixtures library in their tests
       exclude 'com.google.protobuf.**'
+      exclude 'com.google.gson.**'
```

Comment:
🤞  hope this is safe

Reply:
it was the latest change i had to make, the reason is relocation rules didn't result in the same behavior between Java versions and I couldn't make imports work on both, but there is an option to just drop Java 8 compile support so we don't bother to with it, potentially if this is a problem we can drop Java 8

Reply:
how does it work if we just drop java 8, for java 8 consumers?

Reply:
the target byte code version I think will stay 8 for some time, I haven't yet thought in details how we will maintain multiple byte code versions for different consumers, for some internal services modules like tables/jobs I can try produce java17 byte code since the consumer is us